src/json-editor.component.ts

```diff
@@ -290,4 +277,10 @@ export class JsonEditorComponent extends AbstractSubscriberComponent implements
     return element;
   }
 
+  toggleSidebar() {
+    this.isSidebarCollapsed = !this.isSidebarCollapsed;
+    document.querySelector('#expanded-sidebar').classList.toggle('open');
```

Comment:
We never use native api to handle ui unless it's really necessary. This doesn't seem to be one of those cases. The ideal solution for this to make a boolean state (component property) which you already have actually `isSidebarCollapsed`

Then to apply a class on html tag you can bind the boolean

`[class.open]="!isSidebarCollapsed"`

there are also other ways to do it, if you need to apply multiple classes etc.

https://angular.io/api/common/NgClass